app/views/take_whole_pot_calculator/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for(:data_layer) do %>
+  <script>
+    dataLayer = [{
+      'valid': <%= !@result.nil? %>,
```

Comment:
This is knowledge of [how the calculator works](https://github.com/guidance-guarantee-programme/pension_guidance/blob/take-whole-pot-calculator/app/forms/take_whole_pot_calculator_form.rb#L30)... Could also have `@calculator.valid?`